app/src/main/AndroidManifest.xml

```diff
@@ -132,6 +132,7 @@
             android:name=".activities.EventActivity"
             android:exported="true"
             android:label="@string/new_event"
+            android:windowSoftInputMode="adjustResize"
```

Comment:
is this really needed? Updating commons seems to be enough to me as it fixed both EventActivity and TaskActivity

Reply:
Without `adjustResize`, if I try to edit attendee's field, it looks like this:
<img src="https://user-images.githubusercontent.com/36371707/236507578-2c4a839d-81ee-4a45-8eed-1653d4f9bac6.png" width=264 />
The soft input mode defaults to `adjustPan` which hides the toolbar and unnecessarily adds some extra padding at the bottom.

With `adjustResize`, everything looks fine except that the attendee field doesn't scroll up automatically (so it's above the keyboard), one has to do it manually.

Reply:
Resolved the conflicts, if the screenshot above looks okay (not to me) then this PR can be discarded.